rtb/analisi_dei_requisiti_ver0.3.0.typ

```diff
@@ -0,0 +1,325 @@
+
+= Introduzione
+== Scopo del documento
+
+Lo scopo principale del documento Analisi dei Requisiti è quello di fornire una descrizione dettagliata dei casi d'uso e dei requisiti del progetto _NearYou - Smart custom advertising platform_ dell'azienda proponente Sync Lab S.r.l., frutto di uno studio approfondito del capitolato d'appalto e degli incontri con l'azienda proponente stessa.
+
+Il documento risulta di fondamentale importanza in quanto costituisce una chiara visione delle funzionalità e i requisiti attesi riguardo al prodotto e facilita ai fornitori, ovvero il gruppo, nel processo di validazione del prodotto finale.
+
+
+== Scopo del prodotto
+
+Il prodotto _NearYou - Smart custom advertising platform_ è una piattaforma che sfrutta la #rifGlossario("GenAI") per la creazione di pubblicità personalizzate da mostrare a ciascun utente, sfruttando dati come la posizione in tempo reale, le informazioni personali e i dati di profilazione, in modo da migliorare l'esperienza dell'utente e aumentando contemporaneamente il #rifGlossario("ROI") e l'efficacia delle campagne pubblicitarie.
+
+== Glossario
+Per evitare eventuali ambiguità e incomprensioni sulla terminologia adottata nella documentazione redatta dal gruppo,
+viene fornito un Glossario.
+
+La prima occorrenza di un termine definito all'interno del Glossario presente all'interno di un documento viene
+sottolineato e seguito dalla lettera g posta ad apice (e.g. #rifGlossario("termine")).
+
+== Riferimenti
+=== Riferimenti normativi
+- Norme di Progetto
+- Regolamento del progetto didattico\ #formatLink(url: "https://www.math.unipd.it/~tullio/IS-1/2024/Dispense/PD1.pdf")
+- Capitolato C4 (Sync Lab)\ #formatLink(url: "https://www.math.unipd.it/~tullio/IS-1/2024/Progetto/C4.pdf")
+
+=== Riferimenti informativi
+- Glossario
+- T5 - Analisi dei Requisiti\ #formatLink(url: "https://www.math.unipd.it/~tullio/IS-1/2024/Dispense/T05.pdf")
+- P2 - I Diagrammi dei Casi d'Uso\ #formatLink(url: "https://www.math.unipd.it/~rcardin/swea/2022/Diagrammi%20Use%20Case.pdf")
+#pagebreak()
+
+= Descrizione del prodotto
+== Obiettivi del prodotto
+Il prodotto ha come obiettivo quello di fornire un servizio di pubblicità personalizzata integrato in una mappa, accessibile agli utenti di un mezzo a noleggio, che sfrutta la GenAI per la creazione di pubblicità mirate sui singoli utenti. Il gestore del servizio di noleggio avrà accesso ad una _dashboard_ che gli permetterà di monitorare in tempo reale la posizione degli utenti e la visualizzazione degli annunci pubblicitari.
+Per fare ciò l'azienda proponente chiede di simulare lo spostamento degli utenti lungo un percorso, in modo da poter testare il funzionamento del sistema.
+I dati riguardanti gli annunci pubblicitari visualizzati dagli utenti devono essere memorizzati in un _#rifGlossario("database")_ (storicizzazione) in modo da poter essere consultati successivamente dalla _dashboard_, al fine di consentire analisi reportistiche.
```

Comment:
- [ ] qua non è sicuro al 100% che vogliamo fare analisi reportistiche, ci interessa solo storicizzarlo perché l'amministratore li possa vedere. Le analisi è da chiarire perché bisogna vedere quanto tempo ci vuole, e tutto quello che c'è in questo documento poi va fatto, quindi aspetterei, magari mettiamolo come commento.

Reply:
Roba di Martinelli nuovamente. Comunque pensavo fosse richiesto la storia delle reportistiche

Reply:
Non è richiesta, sono sicuro che a Zorzi in particolare sarebbe piaciuta, ma è un di più, dimmi anche tu @davim02

Reply:
ok. Al massimo ritaggatemi e rimuovo

Reply:
> Non è richiesta, sono sicuro che a Zorzi in particolare sarebbe piaciuta, ma è un di più, dimmi anche tu @davim02

Sì per il momento meglio togliere o lasciare commentata quella parte. Mi ricordo di averla scritta quando parlavamo di salvarci il feedback dell'utente. 
Per quanto riguarda invece il salvarsi nel db gli annunci generati non ricordo se fosse richiesto oppure no, forse c'è scritto in qualche verbale... dopo controllo

Reply:
Ok rimuovo solo la roba delle reportistiche e tengo per la dashboard